proposals/memory64/Overview.md

```diff
@@ -6,6 +6,11 @@ This page describes a proposal to support linear memory of sizes larger than
 2<sup>32</sup> bits. It provides no new instructions, but instead extends the
 currently existing instructions to allow 64-bit indexes.
 
+In addition, in order to support source languages with 64-bit pointer width,
+this proposal also extends indirect function tables to allow 64-bit indexes.
```

Comment:
```suggestion
this proposal also extends tables to allow 64-bit indexes.
```

Reply:
I assume we want this for all tables, not just tables containing funcref.

Reply:
Indeed.  Done.